frameworks/kafka/src/main/java/com/mesosphere/sdk/kafka/api/TopicResource.java

```diff
@@ -60,7 +60,15 @@ public Response createTopic(
         try {
             int partCount = Integer.parseInt(partitionCount);
             int replFactor = Integer.parseInt(replicationFactor);
-            return ResponseUtils.jsonOkResponse(cmdExecutor.createTopic(name, partCount, replFactor));
+            JSONObject result = cmdExecutor.createTopic(name, partCount, replFactor);
+            String message = result.getString("message");
+            Response.Status status;
+            if (message.contains("Created topic")) {
+                status = Response.Status.OK;
+            } else {
+                status = Response.Status.INTERNAL_SERVER_ERROR;
```

Comment:
lets  add a  log line here  something like 
` log.error("Failed to verify the topic creation: " + name + " with output: " + message);`

Reply:
Sure @zmalik. I have addressed your requested changes. Thanks.